frontend/src/common/components/dashboard/Dashboard.scss

```diff
@@ -1,21 +1,86 @@
+@import "../../../themes/orbcStyles.scss";
+
 .tabpanel-container {
-  padding: 0 3.75rem;
+  padding: 0 8.551vw;
```

Comment:
we should be consistent with sizing...either `px`, `rem` or `vw`. I don't feel strongly about either one, but if we're aiming to head towards `vw` then let's agree to that -- thoughts @krish1092 ?

Reply:
I agree with being consistent when using either px or rem (I prefer rem), but for this particular scenario the padding size is relative to the viewport width (which is dynamic width) instead of the static size (px or rem).

Reply:
ugh - ok. Not ideal, but I can leave this for now